main.py

```diff
@@ -84,11 +84,20 @@ def img(soup):
 
 def GetPage(path):
     # filter our target path
-    if path[0] != '/':
+    if path[0] != '/' or path in Visited_page:
         return
+    Visited_page.append(path)
     print(path)
```

Comment:
Not sure if this is necessary for keep this printing remain. We can remove it if it don't bring any helpful information or make this printing info more completely as logging message by using logging tool like [loguru](https://github.com/Delgan/loguru), [built-in logging module](https://docs.python.org/3/library/logging.html), etc.